docs/auth/index.md

```diff
@@ -278,3 +289,16 @@ createApiFactory({
     ),
 });
 ```
+
+\_**Warning:** You will need to remove the default `ScmAuth.createDefaultApiFactory();` with this approach.
```

Comment:
Accidental underscore at the start

Reply:
Also maybe go with `**NOTE:** Remember to remove [...]`, warning sounds a bit extreme